corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -26,7 +26,8 @@ FormplayerFrontend.module("Menus", function (Menus, FormplayerFrontend, Backbone
                         FormplayerFrontend.trigger(
                             'showError',
                             response.exception,
-                            response.type === 'html'
+                            response.type === 'html',
+                            true
                         );
```

Comment:
I shied away from this because it felt like breaking some abstraction barrier to change state within the API class